.devcontainer/Dockerfile

```diff
@@ -81,48 +83,74 @@ RUN echo "Install general purpose packages" && \
         tzdata \
         unzip \
         vim \
-        wget && \
-        gem install fpm && \
-        update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-11/bin/clang 10 && \
-        update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-11/bin/clang++ 10 && \
-        update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-7/bin/clang-format 10 && \
-        update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-11/bin/clang-tidy 10 && \
-        update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-11/bin/clang-apply-replacements 10
+        wget \
+    && gem install fpm \
+    && update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-11/bin/clang 10 \
+    && update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-11/bin/clang++ 10 \
+    && update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-7/bin/clang-format 10 \
+    && update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-11/bin/clang-tidy 10 \
+    && update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-11/bin/clang-apply-replacements 10
+
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y clangd-12
```

Comment:
Sorry, that I did not notice earlier. I know you did not touch it, however, in terms of general cleanup, this could also be moved to the list line 44ff, as I see no reason to have it separate. This would also contribute towards layer optimization.